src/controllers/SubmissionSummaryController.ts

```diff
@@ -0,0 +1,20 @@
+import { controller, BaseHttpController, httpGet} from 'inversify-express-utils';
+import {SUBMISSION_SUMMARY_PAGE_URI} from '../utils/Paths';
+
+@controller(SUBMISSION_SUMMARY_PAGE_URI)
+
```

Comment:
Can we remove this new line here, so that the annotation is related to the class declaration?